src/contexts/Api/defaults.ts

```diff
@@ -24,10 +24,6 @@ export const consts: APIConstants = {
 };
 
 export const defaultApiContext: APIContextInterface = {
-  // eslint-disable-next-line
-  connect: async () => {
-    await new Promise((resolve) => resolve(null));
-  },
```

Comment:
I thought that exposing `connect()` in the provider is a bit weird, having `switchNetwork()` exposed as well - and indeed, it's not used anywhere, so removing (this will simplify further de-complication process of the `Api` module).